services/web/server/src/simcore_service_webserver/security_api.py

```diff
@@ -21,7 +21,7 @@
 
 
 def encrypt_password(password: str) -> str:
-    return passlib.hash.sha256_crypt.hash(password, rounds=1000)
+    return passlib.hash.sha256_crypt.using(rounds=1000).hash(password)
```

Comment:
I am always scared of changes in password hashes :-) Let's make sure we have tests of thisone pls